rtusamharam/rtusamharam.json

```diff
@@ -1,5 +1,7 @@
 {
 "title": "ऋतुसंहारम्",
+"custom": {"am": {"name": "ऋतुसमुच्चय (अमरकीर्तिसूरिः)", "lang": "sa", "markdown": true}},
```

Comment:
Could you remove these two for now? These will make the data live immediately. I'll add them after testing it locally.